### ISSUE TITLE
Test server should provide base fixtures

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor5-dev#139]
Right now we need only one time of fixtures – some images. We have a couple of `sample.jpg` in some of our packages to use in manual tests. However, I noticed recently that we have:

```
24 03 2017 17:31:34.177:WARN [web-server]: 404: /foo
```

Logged in automated tests too. That's because we have an integration test of loading an image to the editor.

I propose to configure Karma for automated tests to provide `/sample.jpg` file in all packages. We can add more files in the future. We can also configure the webserver for manual tests to serve such file (and keep in mind that we may want to add more files in the future).

In both cases, the files could be served from `ckeditor5-utils/tests/_assets`. It'd be nice if adding more files in the future were as easy as just copying them there.

I found this regarding Karma: http://stackoverflow.com/questions/30244853/how-to-fix-images-causing-karma-warn-web-server-404

**Comment by ma2ciek** [comment]
Be sure to read the first answer to the related SO question :D

**Comment by Reinmar** [comment]
I assigned it to the current iteration. Seems like an easy fix and will allow us to get rid of all the ugly and confusing 404s in test logs.

**Comment by pomek** [comment]
> We can add more files in the future. We can also configure the webserver for manual tests to serve such file (and keep in mind that we may want to add more files in the future).

Did we have any problem with images or other resources in manual tests? Assets from MT are copied automatically (to the destination directory which is served) while the entire tests are being processed. I prefer to focus on AT and Karma.

> the files could be served from `ckeditor5-utils/tests/_assets`. It'd be nice if adding more files in the future were as easy as just copying them there.

It's doable to configure Karma. All files from the specified directory will be available as `http://0.0.0.0:9876/base/packages/ckeditor5-utils/tests/_assets/[FILE]`. We can also create a proxy:

```js
proxies: {
	'/assets/': '/base/packages/ckeditor5-utils/tests/_assets/'
},
```

All assets can be served now from the `http://0.0.0.0:9876/assets/[FILE]` URL. However, if you would like to remove `/assets` prefix, you need to modify the `proxies` object every time when you need to add a new asset.

We could also look for all files inside the `ckeditor5-utils/tests/_assets` and fill the `proxies` object manually but Karma, in order to serve static files, will do it once again. I guess the prefix is not bad and we could go this way.

**Comment by pomek** [comment]
After adding CKEditor 5 icon (https://github.com/ckeditor/ckeditor5-utils/commit/7059a734c0c0529ec9ca7b129acc1c5d671ded28?short_path=6fe1ae4#diff-6fe1ae49b762910e195699e1e712cb30) and fixing all invalid paths in tests, the entire test log looks much better:

```yarn run v1.16.0
yarn run v1.16.0
$ node --max_old_space_size=4096 node_modules/@ckeditor/ckeditor5-dev-tests/bin/test.js --reporter=dots
ℹ ｢wdm｣:
ℹ ｢wdm｣: Compiled successfully.
ℹ ｢wdm｣: Compiling...
ℹ ｢wdm｣:    2015 modules
ℹ ｢wdm｣: Compiled successfully.
30 07 2019 11:16:42.457:INFO [karma-server]: Karma v4.1.0 server started at http://0.0.0.0:9876/
30 07 2019 11:16:42.460:INFO [launcher]: Launching browsers CHROME_LOCAL with concurrency unlimited
30 07 2019 11:16:42.535:INFO [launcher]: Starting browser Chrome
30 07 2019 11:16:45.123:INFO [Chrome 75.0.3770 (Mac OS X 10.14.4)]: Connected on socket ILomkjQ2HAc7mPLbAAAA with id 98702518
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
................................................................................
.
Chrome 75.0.3770 (Mac OS X 10.14.4): Executed 11041 of 11062 (skipped 21) SUCCESS (2 mins 18.316 secs / 51.217 secs)
Done in 160.08s.
```
